edx_proctoring/tests/test_mfe_views.py

```diff
@@ -0,0 +1,55 @@
+"""
+Tests for the MFE proctored exam views.
+"""
+import json
+from opaque_keys.edx.locator import BlockUsageLocator, CourseLocator
+
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+
+from edx_proctoring.models import ProctoredExam
+
+from .utils import ProctoredExamTestCase
+
+
+User = get_user_model()
+
+
+class ProctoredExamAttemptsMFEViewTests(ProctoredExamTestCase):
```

Comment:
Please add cases for testing active exam as well. 

Both positive and negative scenarios.

Reply:
added, please check